modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -22,11 +24,20 @@ class Test{{classname}}(unittest.TestCase):
     def tearDown(self):
         pass
 
-    def test{{classname}}(self):
+    def make_instance(self, include_optional):
         """Test {{classname}}"""
-        # FIXME: construct object with mandatory attributes with example values
         # model = {{packageName}}.models.{{classFilename}}.{{classname}}()  # noqa: E501
-        pass
+        if (include_optional) :
```

Comment:
Can you remove the parens here? They're not needed

Reply:
Done